dali/operators/input/video_input.h

```diff
@@ -185,10 +197,16 @@ class VideoInput : public VideoDecoderBase<Backend, FramesDecoder>, public Input
   const int device_id_ = {};
   const int batch_size_ = {};
   const std::string last_sequence_policy_;
-
-  /// Valid VideoInput is the one that has the encoded video loaded and will return decoded sequence
-  bool valid_ = false;
-  TensorList<CPUBackend> encoded_videos_;
+  const std::string next_output_data_id_trace_name_ = "next_output_data_id";
```

Comment:
This doesn't need to be an instance member. A static (or a global constant in the .cc file) would do just fine.

Reply:
Done